Packs/BitSight/Integrations/BitSightForSecurityPerformanceManagement/BitSightForSecurityPerformanceManagement.yml

```diff
@@ -0,0 +1,424 @@
+    - contextPath: BitSight.Company.subscriptionType
+      description: subscriptionType
+      type: string
+    - contextPath: BitSight.Company.sparkline
+      description: sparkline
+      type: string
+    - contextPath: BitSight.Company.subscriptionTypeKey
+      description: subscriptionTypeKey
+      type: string
+    - contextPath: BitSight.Company.subscriptionEndDate
+      description: subscriptionEndDate
+      type: string
+    - contextPath: BitSight.Company.bulkEmailSenderStatus
+      description: bulkEmailSenderStatus
+      type: string
+    - contextPath: BitSight.Company.serviceProvider
+      description: serviceProvider
+      type: string
+    - contextPath: BitSight.Company.customerMonitoringCount
+      description: customerMonitoringCount
+      type: string
+    - contextPath: BitSight.Company.availableUpgradeTypes
+      description: availableUpgradeTypes
+      type: string
+    - contextPath: BitSight.Company.hasCompanyTree
+      description: hasCompanyTree
+      type: string
+    - contextPath: BitSight.Company.hasPreferredContact
+      description: hasPreferredContact
+      type: string
+    - contextPath: BitSight.Company.isBundle
+      description: isBundle
+      type: string
+    - contextPath: BitSight.Company.ratingIndustryMedian
+      description: ratingIndustryMedian
+      type: string
+    - contextPath: BitSight.Company.primaryCompany
+      description: primaryCompany
+      type: string
+    - contextPath: BitSight.Company.permissions
+      description: permissions
+      type: string
+    - contextPath: BitSight.Company.isPrimary
+      description: isPrimary
+      type: string
+    - contextPath: BitSight.Company.securityGrade
+      description: securityGrade
+      type: string
+    - contextPath: BitSight.Company.inSpmPortfolio
+      description: inSpmPortfolio
+      type: string
+    - contextPath: BitSight.Company.isMycompMysubsBundle
+      description: isMycompMysubsBundle
+      type: string
+    - contextPath: BitSight.Company.companyFeatures
+      description: companyFeatures
+      type: string
+    description: BitSight command - to get comany details based on guid.
+  - name: bitsight-get-company-findings
+    arguments:
+    - name: guid
+      required: true
+      description: Guid of the company
+    - name: first_seen
+      required: true
+      description: 'First seen date of the findings, Date format is YYYY-MM-DD, Example:
+        2021-01-01'
+    - name: last_seen
+      required: true
+      description: 'Last seen date of the findings, Date format is YYYY-MM-DD, Example:
+        2021-01-01'
+    - name: severity
+      auto: PREDEFINED
+      predefined:
+      - minor
+      - moderate
+      - material
+      - severe
+      description: Minimum Severity of the findings
+    - name: grade
+      auto: PREDEFINED
+      predefined:
+      - good
+      - fair
+      - warn
+      - bad
+      - neutral
+      description: Grade of the findings
```

Comment:
Can the user insert a few values here? if so, please mention in the description this can be a comma-separated list and add all the possible options to the description.